python/python/lance/dataset.py

```diff
@@ -1598,6 +1608,60 @@ def optimize_indices(self, **kwargs):
         self._dataset._ds.optimize_indices(**kwargs)
 
 
+class LanceStats:
+    def __init__(self, dataset: _Dataset):
+        self._ds = dataset
+
+    def dataset_stats(self, max_rows_per_group: int = 1024) -> Dict[str, Any]:
+        return {
+            "count_deleted_rows": self._ds.count_deleted_rows(),
+            "count_fragments": self._ds.count_fragments(),
```

Comment:
Since this is a computed value now, I think these should just be `num_*`:

```suggestion
            "num_deleted_rows": self._ds.count_deleted_rows(),
            "num_fragments": self._ds.count_fragments(),
```

Reply:
Thanks good suggestion! made this change